README.md

```diff
@@ -84,14 +84,18 @@ A Kubernetes Secret also needs to be defined and mounted to the Pod. The secret
 {
   "AuthSecret": "base64 encoded auth secret",
   "S3SecretKey": "s3 secret key",
-  "S3AccessKey": "s3 access key"
+  "S3AccessKey": "s3 access key",
+  "GithubClientID": "github client ID",
+  "GithubClientSecret": "github client secret"
 }
 ```
 
 `AuthSecret` is the secret for authenticating requests. This is needed by the clients to communicate with kontinuous through JWT.
```

Comment:
AuthSecret is still required. the way we integrated github is by wrapping the github access token as JWT afterwards. so auth is still with JWT. this is probably a more flexible approach when we add bitbucket and other scm support.